File: src/test/java/aoc2023/Day04Test.java
package aoc2023;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Day04Test {

    @Test
    public void aoc_example_part1() {
        assertEquals("13", new Day04().asTest().solvePart1());
    }

    @Test
    public void aoc_example_part2() {
        assertEquals("30", new Day04().asTest().solvePart2());
    }
}

File: src/test/java/aoc2025/Day04Test.kt
package aoc2025

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class Day04Test {

    @Test
    fun aoc_example_part1() {
        assertEquals("13", Day04().asTest().solvePart1())
    }

    @Test
    fun aoc_example_part2() {
        assertEquals("43", Day04().asTest().solvePart2())
    }
}
